examples/onnxruntime/inceptionV3/inceptionv3.py

```diff
@@ -0,0 +1,276 @@
+                                   progress=True).eval()
+
+# Download ImageNet labels
+#!curl -o imagenet_classes.txt https://raw.githubusercontent.com/pytorch/hub/master/imagenet_classes.txt
+
+
+def parse_input_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--fp16",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Perform fp16 quantization on the model before running inference',
+    )
+
+    parser.add_argument(
+        "--image_dir",
+        required=False,
+        default="../dataset/images",
+        help='Target DIR for images to infer. Default is ../dataset/images')
+
+    parser.add_argument("--batch",
+                        required=False,
+                        default=1,
+                        help='Batch size of images per inference',
+                        type=int)
+
+    parser.add_argument("--run",
+                        required=False,
+                        default=100,
+                        help='Number of runs for each batched inference.',
+                        type=int)
+
+    parser.add_argument("--top",
+                        required=False,
+                        default=1,
+                        help='Show top K of inference results',
+                        type=int)
+
+    parser.add_argument(
+        "--QPS",
+        action="store_true",
+        required=False,
+        default=False,
+        help=
+        'Show inference result in Queries-Per-Second QPS instead of inference duration (milliseconds)',
+    )
+
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        required=False,
+        default=False,
+        help='Show verbose output',
+    )
+
+    return parser.parse_args()
+
+
+def get_image_list_in_dir(dir):
+    proc = subprocess.run("ls", shell=True, stdout=subprocess.PIPE, cwd=dir)
+    fileList = proc.stdout.decode().split('\n')
+    fileList
+    return fileList
+
+
+def softmax(x):
+    """Compute softmax values for each sets of scores in x."""
+    e_x = np.exp(x - np.max(x))
+    return e_x / e_x.sum()
+
+
+def run_sample(session,
+               categories,
+               latency,
+               inputs,
+               topk,
+               batch_size,
+               verbose=False):
+
+    io_binding = session.io_binding()
+    io_binding.bind_cpu_input('input', inputs.cpu().detach().numpy())
+    io_binding.bind_output('output')
+    start = time.time()
+    session.run_with_iobinding(io_binding)
+    latency.append(time.time() - start)
+    ort_outputs = io_binding.copy_outputs_to_cpu()[0]
+
+    #Get prediction for each item
+    for i in range(batch_size):
+        output = ort_outputs[i].copy().flatten()
+        output = softmax(output)  # this is optional
+        top5_catid = np.argsort(-output)[:topk]
+
+        if verbose:
+            for catid in top5_catid:
+                print(categories[catid], output[catid])
```

Comment:
Print some text here that indicates what it is printing otherwise it will get buried inside verbose logs.